vocabularies/UI.xml

```diff
@@ -1349,7 +1349,7 @@ The trend is
         </Annotation>
       </Term>
 
-      <Term Name="MultiLineText" Type="Core.Tag" Nullable="false" DefaultValue="true" AppliesTo="Property PropertyValue">
+      <Term Name="MultiLineText" Type="Core.Tag" Nullable="false" DefaultValue="true" AppliesTo="Property PropertyValue Parameter">
```

Comment:
```suggestion
      <Term Name="MultiLineText" Type="Core.Tag" Nullable="false" DefaultValue="true" AppliesTo="Property PropertyValue String Parameter">
```
`AppliesTo="Parameter"` is OK, but if it shall apply to a `PropertyValue` (that is, in a complex-typed annotation), then we should also allow `AppliesTo="String"` (for primitive annotations).

Reply:
I don't find `String` in the list of [allowed values for `AppliesTo`](https://docs.oasis-open.org/odata/odata-csdl-xml/v4.01/odata-csdl-xml-v4.01.html#sec_Applicability)

Reply:
If you want to annotate a _term_ of type "String" with `MultiLineText`, then we can add `Term` to the `AppliesTo` list.

Reply:
> I don't find `String` in the list of [allowed values for `AppliesTo`](https://docs.oasis-open.org/odata/odata-csdl-xml/v4.01/odata-csdl-xml-v4.01.html#sec_Applicability)
> If you want to annotate a _term_ of type "String" with `MultiLineText`, then we can add `Term` to the `AppliesTo` list.

But I (as vocabulary author) don't want to annotate the term, the service developer wants to annotate their _usage_ of the term. Means `AppliesTo="Property Parameter PropertyValue Annotation"`?

Reply:
Yes, if you only want to annotate the usage of a term, then `Annotation` is the correct value.

Do we have use cases for annotating annotations with this term, or can we wait until such use cases pop up?

Reply:
```
<Annotation Term="Core.Description" String="A very long description">
  <Annotation Term="UI.MultiLineText"/>
</Annotation>
```
seems very natural to me.